lib/aws/s3.js

```diff
@@ -93,7 +124,33 @@ export class S3Client {
         ...params
       }
 
-      const res = await this._s3.getObject(req).promise()
+      const data = await this._s3.getObject(req).promise()
+
+      log.debug({ data }, 'data')
+      log.info('end')
+      return data.Body
```

Comment:
We need the ContentType as well, so we need the entire object back or similar.

Reply:
Ah right passing back the raw s3 resp